stix_shifter_utils/stix_transmission/utils/RestApiClient.py

```diff
@@ -11,15 +11,21 @@
 
 # This is a simple HTTP client that can be used to access the REST API
 
-RETRY_MAX = 1
+RETRY_MAX_DEFAULT = 1
+CONNECT_TIMEOUT_DEFAULT = 2
 
 
 class RestApiClient:
     # cert_verify can be
     #  True -- do proper signed cert check that is in trust store,
     #  False -- skip all cert checks,
     #  or The String content of your self signed cert required for TLS communication
-    def __init__(self, host, port=None, headers={}, url_modifier_function=None, cert_verify=True,  sni=None):
+    def __init__(self, host, port=None, headers={}, url_modifier_function=None, cert_verify=True,  sni=None, auth=None):
+        self.retry_max = os.getenv('STIXSHIFTER_RETRY_MAX', RETRY_MAX_DEFAULT)
+        self.retry_max = int(self.retry_max)
+        self.connect_timeout = os.getenv('STIXSHIFTER_CONNECT_TIMEOUT', CONNECT_TIMEOUT_DEFAULT)
+        self.connect_timeout = int(self.connect_timeout)
+
```

Comment:
@yurii-klymenko where does this env variable specified -`STIXSHIFTER_RETRY_MAX` , `STIXSHIFTER_CONNECT_TIMEOUT`

Reply:
they are not, they are optional